File: src/main/java/tr/com/ias/entity/iasHtmlLabel.java
package tr.com.ias.entity;

public class iasHtmlLabel extends iasHtmlComponent {

	private String text;

	public iasHtmlLabel() {
		super();
		componentType = ComponentType.LABEL;
	}

	public iasHtmlLabel(String text) {
		this();
		this.text = text;
	}

	public String getText() {
		return text;
	}

	public void setText(String text) {
		this.text = text;
	}

}

File: front_end/dynamic-component/src/app/ias-html-label.ts
import { iasHtmlComponent } from './ias-html-component';
import { ComponentType } from './component-type';


export class iasHtmlLabel extends iasHtmlComponent{
	text: string;
	  constructor(e: JSON) {
	  	super();
		  this.componentType = ComponentType.LABEL;
		  this.x = e['x'];
		  this.y = e['y'];
		  this.z = e['z'];
		  this.caption = e['caption'];
		  this.text = e['text'];
	   }

}